mlflow/models/model.py

```diff
@@ -550,8 +559,15 @@ def log(
             local_path = tmp.path("model")
             run_id = mlflow.tracking.fluent._get_or_start_run().info.run_id
             mlflow_model = cls(artifact_path=artifact_path, run_id=run_id, metadata=metadata)
-            flavor.save_model(path=local_path, mlflow_model=mlflow_model, **kwargs)
+            flavor.save_model(
+                path=local_path, mlflow_model=mlflow_model, signature=signature, **kwargs
+            )
             mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
+            tracking_uri = _resolve_tracking_uri()
+            if (
+                tracking_uri == "databricks" or get_uri_scheme(tracking_uri) == "databricks"
+            ) and signature is None:
+                _logger.warning(_LOG_MODEL_MISSING_SIGNATURE_WARNING)
```

Comment:
Q: Why it only prints warning on databricks ?

Reply:
We wanted to limit the blast radius of this alert. Didn't want to spam non-Databricks customers with warnings about model signatures.

Reply:
Do you feel like it should be unlimited?

Reply:
@harupy What do you think ? I incline to make OSS and databricks printing consistent warnings

Reply:
> I incline to make OSS and databricks printing consistent warnings

Why?

Reply:
Just warning on Databricks sounds ok to me.

Reply:
`upcoming model registry features` refers to Databricks features, right?

Reply:
> upcoming model registry features refers to Databricks features, right?

Yes